src/data-providers/firebase.ts

```diff
@@ -94,4 +112,53 @@ const getFirebaseProvider = (state: State, dispatch: Dispatch<any>) => ({
   },
 })
 
+/** Creates a subscription handler that converts a Parent snapshot to a ThoughtUpdate and invokes a callback. */
+const createParentSubscriptionHandler =
+  (onUpdate: (updates: ThoughtUpdates) => void, { value }: { value?: Parent | null } = {}) =>
```

Comment:
We can replace `value`  with something like `delete` which would be a boolean. If `delete` is true, then simply use `null` as value. I think it would be more clear to understand.

Reply:
I tried it both ways, and I find `{ value: null }` to read better than an unmarked Boolean. I'll add a comment though. The Boolean would require a comment anyway to understand.